README.md

```diff
@@ -21,6 +21,17 @@ $ npm install --save resin-register-device
 Documentation
 -------------
 
+Resin-Register-Device exports a factory function, which must be called with a dependencies object, containing a configured [`resin-request`](https://github.com/resin-io-modules/resin-request) instance.
```

Comment:
As I've commented in the other PR I'm for switching this for receiving opts and instantiating `resin-request` on its own. At the very least just because other 6 or 7 modules behave like that already, and (just repeating) because it's in line with normal npm deps usage.